.travis.yml

```diff
@@ -2,7 +2,9 @@ language: java
 sudo: false
 
 script:
-  - mvn --batch-mode clean install
+  - xvfb-run mvn --batch-mode clean install -Dmaven.test.skip=false 
```

Comment:
xvfb-run is cool! But instead of `-Dmaven.test.skip=false` here, how about just [removing the skip hard-coded in all POMs](https://github.com/m2e-code-quality/m2e-code-quality/search?l=Maven+POM&q=skip)? That way, people also run these tests locally (if we do it like proposed here, nobody will ever look at .travis.yml, and there will be confusion why the Travis build is different from the local build)